src/core/sent_packet_metadata.h

```diff
@@ -94,6 +94,7 @@ typedef struct QUIC_SEND_PACKET_FLAGS {
     //
     BOOLEAN IsAppLimited            : 1;
     BOOLEAN HasLastAckedPacketInfo  : 1;
+    BOOLEAN EcnEctSet               : 1;
```

Comment:
This makes us use another byte per packet. I wonder if there's a way to eliminate this need.

Reply:
That's not possible I think. We need to know which packets are sent with ECT bit set.

Are you sure this will use another byte? I don't see clear effort for struct packing in QUIC_SENT_PACKET_METADATA.

Reply:
I validated the size in the past. I can't claim 100% certainty now.

Reply:
Did you validate on Release or Debug?

Reply:
I validated it on both. It's 80B regardless of build flavor.